book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/0_5_0_laptop.md

```diff
@@ -184,7 +184,7 @@ Follow [these instructions](https://docs.docker.com/docker-for-mac/install/)
 ### Duckietown Shell {#laptop-setup-mac-shell}
 
 Follow [these instructions](https://github.com/duckietown/duckietown-shell)
-
+Note: On OS X if you get a `Could not load commands` error when you run `dts` for the first time. Try deleting `~/.dt-shell/commands-multi/daffy/challenges/installed.flag` and running `dts` again.
```

Comment:
Thanks for the Note. I believe the ` character needs to be escaped in the "MarkDuck" format. Or you could switch it to another type of quote.

Reply:
It was my intent to have the ` quoted parts to appear like `/usr/X11/bin` and `xhost` in the note above this note in the same file that use ` to change the style. 
Is this correct way of doing things?